src/components/menuPage/FilterTagBar.js

```diff
@@ -0,0 +1,30 @@
+
+
+const FilterTagBar = () => {
+let activeHandler=(e)=>{
+    let activeList = document.getElementsByClassName('active')
+    if(activeList.length!==0){
+        (activeList)[0].classList.remove('active');
+    }
+        e.currentTarget.className += ' active'
+}
+
+
+  return (
+    <div id='scrollTag' className="scrollTag sticky-top bg-white fw-semibold fs-5 d-flex justify-content-lg-center">
+
+      <a className="scrollItem" href="#pizzas" onClick={activeHandler}>
```

Comment:
I think we can use React Link here instead of anchor. Could you take a look?

Reply:
For here, It just targets the id value and jumps to the specific part on the same page. React link is router-level, so I don't think we need to use React Link.

Reply:
Ok